features/step_definitions/event_steps.rb

```diff
@@ -255,6 +255,21 @@ def stub_user_browser_to_specific_timezone
   @google_id = '123456789'
 end
 
+
+When(/^the HangoutConnection has pinged to indicate the event start, an appropriate tweet will be sent$/) do
+  yt_tweet = "Alejandro just hosted an online #scrum Missed it? Catch the recording at youtu.be/11 #CodeForGood #opensource"
+  hangout_tweet = "#Scrum meeting with our #distributedteam is live on http://hangout.test Join in and learn about our #opensource #projects!"
+  expect(TwitterService.twitter_client).to receive(:update).with(hangout_tweet).and_call_original
```

Comment:
could we be checking that the VCR cache was hit correctly?  that would tie us to the implementation less ...

Can we use a RSpec spy to allow us to have separate steps and put them in an order that is more readable?

Or should we just leave that and move on ...?